src/styles/Navbar.css

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: grey;
+  padding: 1rem 2rem;
+  margin: 0 auto;
+  box-shadow: 0 4px 14px 0 rgba(70, 70, 70, 0.38);
+}
```

Comment:
- [ ] 👋 Hello! Great work on this milestone! 🎉 However, I've noticed a small issue with the current implementation, particularly regarding web responsiveness. 📱 It seems that the navbar and page containers are overflowing, causing a less-than-optimal user experience on small-screen devices like mobile. 🙁

![Screenshot from 2023-09-18 17-17-51](https://github.com/blessing-michael/Math-Magicians/assets/112261237/f74a72bc-81eb-45e2-8c74-95a3879d61cd)

Certainly! Let's provide more insights and suggest some solutions to address the web responsiveness issues:

📱 Mobile-Friendly Magic Needed! 🪄

The world is shifting towards mobile-first design, and ensuring your web application is responsive is crucial. Here's how we can improve it:

1. Hamburger Menu for Mobile: 🍔

On smaller screens like mobile devices, the navbar can be transformed into a hamburger menu. This collapsible menu icon, typically represented by three horizontal lines, is a common and user-friendly way to handle navigation on mobile. When tapped, it expands to reveal the menu items.

2. Responsive CSS Grids or Flexbox: 📏

To prevent overflowing and ensure content adapts to various screen sizes:

Utilize CSS grids or flexbox to create responsive layouts for your page containers. These CSS techniques allow you to design fluid and adaptable structures that work well on both large and small screens.

3. Media Queries for Custom Styling: 🖥️📱

Implement media queries in your CSS to apply custom styling based on screen size. For example:
```
@media screen and (max-width: 768px) {
  /* Add styles for screens up to 768px wide (typical tablet and mobile sizes) */
}

@media screen and (max-width: 480px) {
  /* Add styles for screens up to 480px wide (typical mobile sizes) */
}
```
This allows you to make specific adjustments for different device widths.

4. Testing Across Devices: 🧪

After making these changes, it's crucial to thoroughly test your web application on various devices and screen sizes. Use device emulators, browser developer tools, or physical devices to ensure everything looks and functions as expected.

By implementing these solutions, you can achieve a more responsive and user-friendly design across all devices, including mobile. Remember that user experience should always be a top priority in web development! 😊🚀

Reply:
[zeff96](https://github.com/zeff96) Thank you for the observation, However I will like to bring to your notice that web responsivesness is not one of the project requirement. I have met the requirements and this review is blocking my moving forward to the next activity. I will add the mentioned feature in future if needed as the project requirement.  However now, I will like to move to the next activity. kindly look into this review again. Thank you.

Reply:
👋 Hello! Great work on this milestone! 🎉 However, upon consulting with a senior reviewer, I've learned that it was not necessary to request these changes as a mandatory requirement. I sincerely apologize for any inconvenience caused.

Thank you for your understanding. If you have any further questions or need assistance with anything else, please feel free to let me know